hooks/_common.sh

```diff
@@ -226,7 +245,10 @@ function common::per_dir_hook {
   # run hook for each path
   for dir_path in $(echo "${dir_paths[*]}" | tr ' ' '\n' | sort -u); do
     dir_path="${dir_path//__REPLACED__SPACE__/ }"
-    pushd "$dir_path" > /dev/null || continue
+
+    if ! $SKIP_CHDIR; then
```

Comment:
Given you define `SKIP_CHDIR` to a non-empty value on line 221 (`local SKIP_CHDIR=false`) the `! $SKIP_CHDIR` would always be false since Bash has no idea about boolean values. Hence need to compare strings explicitly:
```suggestion
    if [[ $SKIP_CHDIR != true ]]; then
```